scripts/hazard_process.py

```diff
@@ -66,6 +66,72 @@ def process_national_hazard(country):
     return
 
 
+def convert_national_hazard(country):
+    """
+    This function converts the national hazard .tif file into
+    a .shp file
+    
+    """
+    iso3 = country['iso3']
+    gid_region = country['gid_region']
+
+    folder_out = os.path.join('data', 'processed', iso3 , 'hazards', 'inuncoast', 'national')
+    if not os.path.exists(folder_out):
+        os.makedirs(folder_out)
+    filename = 'inuncoast_rcp8p5_wtsub_2080_rp1000_0.shp'
```

Comment:
This works for now, but at some point we will need to introduce the climate scenarios, and return periods here, which means we need a loop.